src/components/video-embed/helpers/video-hooks-test-component.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect } from 'react'
+import {
+  usePlayState,
+  useMilestones,
+  useSecondsWatched,
+  useSegmentsPlayed,
+} from '.'
+
+const PROGRESS_INTERVAL = 1000
+const MAX_PLAYBACK_SPEED = 2.0
+
+/**
+ * This component is for testing purposes only.
```

Comment:
It's tough to test video playback in Jest due to what I _think_ are some inherent limitations (the embedded video seems to not render at all).

This component makes it possible to run some similarly high-level tests on our hooks (which do most of the work outside of `react-player`), at the cost of having to manually implement the playback events we expect from `react-player` in the main `VideoEmbed` component.